tests/MongoDB.Driver.Core.Tests/Specifications/connection-string/ConnectionStringTestRunner.cs

```diff
@@ -146,6 +146,9 @@ private void AssertOptions(ConnectionString connectionString, BsonDocument defin
                         case "tlsinsecure":
                             AssertBoolean(connectionString.TlsInsecure, expectedOption.Value);
                             break;
+                        case "tlsdisablecertificaterevocationcheck":
```

Comment:
Just slightly out of alphabetical order... :)

Reply:
Doh! :) Fixed